pkgs/development/python-modules/crytic-compile/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, buildPythonPackage, fetchPypi, makeWrapper, pythonOlder
+, setuptools
+, pysha3
+, solc
+}:
+
+buildPythonPackage rec {
+  pname = "crytic-compile";
+  version = "0.1.6";
+
+  disabled = pythonOlder "3.6";
+
+  # No Python tests
+  doCheck = false;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "32c30a9c4e6d2122fe01b21648127bca949c43402ed5a91c10bf3aa34a53731a";
+  };
+  
+  nativeBuildInputs = [ makeWrapper ];
+  propagatedBuildInputs = [ pysha3 setuptools ];
+
+  postFixup = ''
+    wrapProgram $out/bin/crytic-compile \
+      --prefix PATH : "${lib.makeBinPath [ solc ]}"
+  '';
+
+  meta = with lib; {
+    description = "Abstraction layer for smart contract build systems";
+    longDescription = ''
+      Crytic-compile is a library to help smart contract compilation.
+      It is used in many Crytic tools, e.g. Slither.
+    '';
+    homepage = https://github.com/crytic/crytic-compile;
```

Comment:
The URL should be double quoted, the URL type is going away.

Reply:
> @jerengie thanks, this looks good! (modulo the changes outlined by @jonringer)
> Would you consider adding yourself to the maintainers? I'm not using this anymore.

Thank you. Sure, I now added myself to the list of maintainers for `slither-analyzer`.